### ISSUE TITLE
Data files not being packaged

### ISSUE THREAD

**Issue by ahwagner** [ga4gh/vrs-python#48]
It appears that the schema files from vr-spec (both the revised `vr.*` and the `ga4gh.*` files) are not getting packaged. Not sure yet why this isn't working.

**Comment by reece** [comment]
Confirmed.

Travis deployments have stopped including the schema files for vrs.

**Background**

The vrs-python repo includes vr-spec as a submodule. 

The intention is that VRS schema files (i.e., vr-spec/schema/ga4gh.{yaml,json} and vrs.{yaml,json} are included the Python package. That means that each vrs-python package release is effectively bound to a particular release of vrs.

The layout is like this:
```
snafu$ ls -ld  src/ga4gh/*/_internal/data/schema vr-spec/schema/
lrwxrwxrwx 1 reece reece   29 Dec 26  2019 src/ga4gh/core/_internal/data/schema -> ../../../../../vr-spec/schema/
lrwxrwxrwx 1 reece reece   29 Sep 26 09:15 src/ga4gh/vrs/_internal/data/schema -> ../../../../../vr-spec/schema/
drwxr-xr-x 2 reece reece 4.1k Jan  4 10:23 vr-spec/schema//
```

`setup.cfg` declares _internal/data/schema paths for each Python submodule (ga4gh.core and ga4gh.vrs) to be included in the Python package. The modules then read these at runtime using `pkg_resources`.

For the `0.5.0.post1` release in Sep 2020 (the last release), [Travis shows this](https://travis-ci.org/github/ga4gh/vrs-python/jobs/727251174#L609):
```
adding 'ga4gh/core/_internal/data/schema/ga4gh.json'
adding 'ga4gh/core/_internal/data/schema/ga4gh.yaml'
adding 'ga4gh/vr/__init__.py'
adding 'ga4gh/vr/dataproxy.py'
adding 'ga4gh/vr/normalize.py'
adding 'ga4gh/vr/_internal/enderef.py'
adding 'ga4gh/vr/_internal/models.py'
adding 'ga4gh/vr/_internal/data/schema/vr.json'
adding 'ga4gh/vr/_internal/data/schema/vr.yaml'
```

[Today's release shows](https://travis-ci.org/github/ga4gh/vrs-python/jobs/753256385) only the files for core:
```
adding 'ga4gh/core/_internal/data/schema/ga4gh.json'
adding 'ga4gh/core/_internal/data/schema/ga4gh.yaml'
```

I have not deduced the underlying cause yet.

**Comment by reece** [comment]
As far as I can tell, nothing has changed about the symlink structure since the last build.

On my local machine, `make bdist_wheel` builds the wheel package correctly. I now suspect that there's a subtle change in the packaging mechanics at travis that is causing this bug. :-(

**Comment by reece** [comment]
Fixed in 6da3ba0. Underlying cause was that schema name changed from vr.json to vrs.json in the development branch.
